07-http_gordle/5_play/internal/api/convert.go

```diff
@@ -1,19 +1,19 @@
-package apiconversion
+package api
 
 import (
-	"learngo-pockets/httpgordle/api"
 	"learngo-pockets/httpgordle/internal/session"
 )
```

Comment:
similarly, extra brcakets here (that were removed in the other convert.go files)